db/indexes.go

```diff
@@ -175,9 +246,10 @@ func waitForIndexes(bucket *base.CouchbaseBucketGoCB, useXattrs bool) error {
 			indexesWg.Add(1)
 			go func(index SGIndex) {
 				defer indexesWg.Done()
-				queryStatement := replaceSyncTokens(index.readinessQuery, useXattrs)
+				queryStatement := replaceSyncTokensQuery(index.readinessQuery, useXattrs)
```

Comment:
It might be helpful during debugging if there was an additional `LogTo("Index+")` log here that said "Waiting for index readiness for index foo" and "Done waiting for index readiness for index foo"